### PR TITLE
Add NodeJS env

### PR DESCRIPTION
**Pull request by emrullahcirit**
Installation of NodeJS environment with pacman.

### CODE REVIEW

**Comment by cagataycali** [comment]
LGTM

**Review by cagataycali — DISMISSED** [review]

File: updates/2_add_nodejs_env.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  exit 444
+}
+
+success() {
+  echo "$*" >&2
+  exit 0
+}
+
+main() {
+  pacman -S nodejs npm
```
Comment (boratanrikulu) [inline updates/2_add_nodejs_env.sh]: You should add --no-confirm tag. The user already accepted the installation on the update system.
Update it like this:
`pacman -S --no-confirm nodejs npm`

**Review by boratanrikulu — CHANGES_REQUESTED** [review]
Add --no-confirm tag

File: updates/2_add_nodejs_env.sh (hunk 1 above)
Comment (emrullahcirit) [reply updates/2_add_nodejs_env.sh]: noconfirm option added.

**Review by boratanrikulu — APPROVED** [review]

**Pull request merged**